lib/dialogs/import/dropzone/index.tsx

```diff
@@ -107,17 +89,22 @@ function ImporterDropzone({
         {file.name}
       </li>
     ));
+
+    const errorList = errorMessage.map((error) => (
+      <div key={error}>
+        <WarningIcon />
+        {error}
```

Comment:
I don't know if it matters but if it's a list we might consider generating that HTML for the sake of screenreaders. it's not like this app is in a good shape for accessibility, but if we're adding new code we might be able to help.

```ts
{errorMessages.length && (
	<ul>
		{errorMessages.map(error => (
			<li key={error}><WarningIcon />{error}</li>
		)}
	</ul>
)}
```

Reply:
You're right, I'll change it back to a list. I had it that way at first and then changed to divs temporarily to avoid some styling issues while making sure it worked.

Reply:
I'm terrible with styles. when I build something I toss in some garbage and ask @belcherj to make it pretty. he's quite good at that.